src/wui/info_panel.cc

```diff
@@ -65,7 +65,7 @@ MessagePreview::MessagePreview(InfoPanel* i, const Widelands::Message* m, Widela
 }
 
 inline bool MessagePreview::message_still_exists() const {
-	return !id_ || (owner_.message_queue_ == nullptr) ||
+	return !(id_.operator bool()) || (owner_.message_queue_ == nullptr) ||
```

Comment:
why not use id == 0 or something like we did in the other instances

Reply:
`id_` is a class type: https://github.com/widelands/widelands/blob/master/src/logic/message_id.h
The class defines a bool conversion operator to check if an instance denotes a valid ID. If such an operator exists then it must be used, hardcoding the "invalid" constant as 0 (and not e.g. 0xffffffff) would complicate changing the backend in the future.